emop-maven-plugin/src/main/java/edu/cornell/VmsMojo.java

```diff
@@ -228,6 +228,8 @@ private boolean filter(String violation) {
     private void invokeSurefire() throws MojoExecutionException {
         InvocationRequest request = new DefaultInvocationRequest();
         request.setGoals(Collections.singletonList("surefire:test"));
+        request.setProperties(System.getProperties());
+        request.setBatchMode(true);
```

Comment:
Out of curiosity - what exactly was the problem that surefire was running into with the extension / how does this fix it?

Reply:
From what I've observed, the problem wasn't with the extension. It was that some properties were being set on the command line (e.g., `maven.repo.local`) and these weren't being passed to Surefire. This change just forwards all the properties that VMS was run with to Surefire.